lib/sass.dart

```diff
@@ -33,3 +33,23 @@ String render(String path,
       evaluate(sassTree, color: color, packageResolver: packageResolver);
   return toCss(cssTree);
 }
+
+/// Loads [source] as Sass, converts it to CSS, and returns the result.
```

Comment:
"as Sass" makes it a little ambiguous whether this is parsed as the indented syntax or as SCSS by default. I think SCSS is the right answer, but it should be clearer in the docs. It would also be good to add an option to configure that.